src/components/attribute-input/AttributeInput.tsx

```diff
@@ -58,63 +58,95 @@ export const AttributeInput = ({
     }
   }, []);
 
-  const [isOpenArray, setIsOpenArray] = useState<boolean[]>([false]);
+  const [isKeyOpenArray, setIsKeyOpenArray] = useState<boolean[]>([false]);
   const watchLastKey = watch(`${name}[${fields.length - 1}].key`, "");
   const watchLastValue = watch(`${name}[${fields.length - 1}].value`, "");
 
-  const [valueOpen, setValueOpen] = useState(false);
-  const toggleSelect = (rowIndex: number, open: boolean) => {
-    const arr = [...isOpenArray];
+  const [isValueOpenArray, setIsValueOpenArray] = useState<boolean[]>([false]);
+  const toggleKeySelect = (rowIndex: number, open: boolean) => {
+    const arr = [...isKeyOpenArray];
+    arr[rowIndex] = open;
+    setIsKeyOpenArray(arr);
+  };
+
+  const toggleValueSelect = (rowIndex: number, open: boolean) => {
+    const arr = [...isValueOpenArray];
     arr[rowIndex] = open;
-    setIsOpenArray(arr);
+    setIsValueOpenArray(arr);
   };
 
   const renderValueInput = (rowIndex: number, attribute: any) => {
-    const attributeValues = defaultContextAttributes.find(
-      (attr) => attr.key === attribute.key
-    )?.values;
+    let attributeValues = [] as
+      | {
+          key: string;
+          name: string;
+        }[]
+      | undefined;
```

Comment:
I think you can remove the type cast here and use an annotation instead.
```suggestion
    let attributeValues: { key: string; name: string; }[] | undefined = [];
```